### PR TITLE
chore(pkgs/poetry): 1.1.11 -> 1.2.0a2

### PR DESCRIPTION
**Pull request by lovesegfault**

### CODE REVIEW

File: pkgs/poetry/pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "poetry"
-version = "1.1.11"
+version = "1.2.0a2"
```
Comment (l0b0) [inline pkgs/poetry/pyproject.toml:3]: Why upgrade to a [testing release](https://python-poetry.org/blog/announcing-poetry-1.2.0a2/)?

File: pkgs/poetry/update [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env nix-shell
 #! nix-shell -i bash -p curl nix-prefetch-github jq
 
-rev=$(curl -s https://api.github.com/repos/python-poetry/poetry/releases/latest | jq -r '.name')
+rev=1.2.0a2
```
Comment (l0b0) [inline pkgs/poetry/update:4]: Doesn't this defeat the purpose of this script?

Comment (lovesegfault) [reply pkgs/poetry/update:4]: I shouldn't have added that to the commit, it was a oneshot thing.

File: pkgs/poetry/pyproject.toml (hunk 1 above)
Comment (lovesegfault) [reply pkgs/poetry/pyproject.toml:3]: It's not meant to be merged now, just so that we can start playing with it

**Pull request closed without merging**